### DISCUSSION TITLE
Change the glint

### DISCUSSION THREAD

**Discussion by E-Nic95** [BedrockTweaks/Files#332] [Ideas]
You know, I feel you could change the glint for when the crops are ready, I mean, you can't make the 3D exclamation mark, but why not make it part of the texture? I modified the glint for personal purposes And changed it to the exclamation mark, and it works! I also changed the flipbook script (or whatever) so that It's like the real thing (almost) if you want to keep the glint but add the exclamation mark, just make it two separate packs, I know how to use blockbench and figured out how the vanilla tweaks exclamation mark works, lotta text lol.

**Comment by PAPISOP** [comment]
> You know, I feel you could change the glint for when the crops are ready, I mean, you can't make the 3D exclamation mark, but why not make it part of the texture? I modified the glint for personal purposes And changed it to the exclamation mark, and it works! I also changed the flipbook script (or whatever) so that It's like the real thing (almost) if you want to keep the glint but add the exclamation mark, just make it two separate packs, I know how to use blockbench and figured out how the vanilla tweaks exclamation mark works, lotta text lol.

I agree and I don't want to replace the shine indicator (many people might like it till now) so let's just add 2 new packs which adds an arrow indicator 
*you can contribute and make a pr if this suggestion gets accepted*

**Reply by E-Nic95** [reply]
I already make a resource pack, and uploaded it to MCPEDL, giving credits of course, the pack is still in pending at the moment of writing this.
